internal/db/read_writer_test.go

```diff
@@ -3193,3 +3193,16 @@ func TestDb_oplogMsgsForItems(t *testing.T) {
 // 		})
 // 	}
 // }
+
+func TestDb_TransactionTimestamp(t *testing.T) {
```

Comment:
Can we add tests for calling `TransactionTimestamp` multiple times within the same transaction?

Reply:
1. Multiple calls to `TransactionTimestamp` within the same transaction should return equal `time.Time` values (might want to sleep for a second between calls).
2. Multiple calls to `TransactionTimestamp` not in a transaction should return values of `time.Time` that are not equal (again, might want to sleep for a second between calls).